src/explore-education-statistics-admin/src/routes/publicationRoutes.ts

```diff
@@ -50,3 +58,21 @@ export const publicationContactRoute: PublicationRouteProps = {
   title: 'Contact',
   component: PublicationContactPage,
 };
+
+export const publicationLegacyReleasesRoute: PublicationRouteProps = {
+  path: '/publication/:publicationId/legacy',
+  title: 'Legacy releases',
+  component: PublicationLegacyReleasesPage,
+};
+
+export const publicationCreateLegacyReleaseRoute: PublicationRouteProps = {
+  path: '/publication/:publicationId/legacy/create',
+  title: 'Create Legacy Release',
```

Comment:
A little weird to capitalise every word. We typically don't do this in other places (e.g. `releaseRoutes`).

Would change this to 'Create legacy release'.